Examples/test-suite/php/cpp11_attribute_specifiers_runme.php

```diff
@@ -3,11 +3,26 @@
 require "tests.php";
 
 // New functions
-check::functions(array('noReturn','noDiscard','noDiscardDeprecated','maybeUnused1','maybeUnused2','likely','test_string_literal'));
+check::functions(array(
+  '_noret_noReturn',
+  '_noDiscard',
+  '_noDiscardDeprecated',
+  'maybeUnused1',
+  'maybeUnused2',
+  '_likely',
+  'test_string_literal',
+  'aligned_var_1_set',
+  'aligned_var_1_get',
+  'aligned_var_2_set',
+  'aligned_var_2_get',
```

Comment:
These getters and setters should get automatically ignored as a result of `check::globals(array('aligned_var_1', 'aligned_var_2'));`.